src/ports/postgres/modules/utilities/path.py_in

```diff
@@ -108,8 +107,11 @@ def path(schema_madlib, source_table, output_table, partition_expr,
                             *,
                             nextval('{seq_gen}') AS {id_col_name},
```

Comment:
the sequence should be more efficient in this case. We only need a unique id for each row, it doesn't have to be gapless - so the sequence would be better.